docs/mindsdb-docs/docs/sql/tutorials/process-quality.md

```diff
@@ -22,18 +23,31 @@ In this tutorial you will learn how to predict the quality of a mining process u
 1. Fix headers: 
    - `sed -e 's/ /_/g' -e 's/\(.*\)/\L\1/' -e 's/%_//g' MiningProcess_Flotation_Plant_Database.csv > fixed_headers.csv` (for Linux/Unix)
    - edit headers manually: change `space` to `underscore`, upper case to lower case, remove `%` from headers (for Windows)
-2. Click on **Files** icon to go to datasets page
-3. Click on **FILE UPLOAD** button to upload file into MindsDB
+2. Access MindsDB GUI either on cloud or local via URL 127.0.0.1:47334/.
+3. Select the `Add data` button or the plug icon on the left side bar.
+4. The page will navigate to the 'Select your data source' page. Select 'Files'.
+5. Under 'Import a file' select the tab 'Click here to browse local files' and select your data file.
+6. Once the file is uploaded 100%, provide a name for the data file that will be saved as a table.
+7. Select the button Save and Continue.
 
+You can query the file that has been uploaded to see that the data does pull through.
+
+```sql
+SELECT * from files.file_name;
+```
 
 ## Connect to MindsDB SQL Sever
+
+MindsDB's GUI has a SQL Editor that allows you to create and train predictors and also makes predictions. However you can also do this via the mysql client in a local terminal by accessing MindsDB SQL Server.
+
 1. 
 ```sql
 mysql -h cloud.mindsdb.com --port 3306 -u username@email.com -p
 ```
 2. 
 ```sql
 USE mindsdb;
+```dsdb;
```

Comment:
Do we need this?

Reply:
That's just for the people who are using mysql client instead of the GUI. Cause we have enough tutorials on how to do it in the SQL Editor,and there might be users  that use the client like the one of the users I have been chatting to... the 'dsdb needs to be taken out